test/functional/comment_controller_test.rb

```diff
@@ -42,6 +42,29 @@ def teardown
     assert_template partial: 'questions/_comment'
   end
 
+  test 'should create wiki comments' do
+    UserSession.create(rusers(:bob))
+    assert_difference 'Comment.count' do
```

Comment:
Can we add one more assert_difference here, to verify that `node(:wiki_page).comments.length` changes by 1? There are other examples in functional tests for testing difference by a given quantity. That way we'll know that the comment is not only created but that it's attached to the wiki record.